package.json

```diff
@@ -41,7 +41,7 @@
     "@octokit/types": "^13.5.0",
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "@types/react-infinite-scroller": "^1.2.5",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/eslint-plugin": "^8.8.0",
     "@typescript-eslint/parser": "^5.56.0",
```

Comment:
- `@typescript-eslint/eslint-plugin` has a dependency stated just below which is `@typescript-eslint/parser`.
- Bumping version failed because the dependency version was not changed.